### ISSUE TITLE
filetest unneccessarily dependency of filetest on doubleword extension

### ISSUE THREAD

**Issue by mitra42** [gerryjackson/forth2012-test-suite#27]
The filetest.sh tests contain double words for example `0.` or `10.` and a couple of other places like 2VARIABLE which are only available in the Double Number word set.

I believe I have a fully compliant File-Access word set in webforth, but I can't prove this because I haven't (yet) implemented the Double Number word set. 

I added the following line (immediately under the "DECIMAL" to get it to work, but there might be a way more appropriate to your style.

: D= ROT = -ROT = AND ; : 2VARIABLE VARIABLE 2 ALLOT ; : 0. 0 0 ; : 50. 50 0 ; : 500. 500 0 ; : 10. 10 0 ; : 37. 37 0 ;

**Comment by gerryjackson** [comment]
Thank you for reporting the dependency on the Double Number word set. I have changed file-test.fth to remove the dependency. I've implemented the changes differently:
1. Defined DEQ to avoid conditional compilation for those systems that have D=, also -ROT isn't a standard word.
2. Similarly 2VARIABLE has been replaced by CREATE FP 0 , 0 , for the same reason
3. Double numbers such as 500. are replaced by 500 0 because it's simpler

Thanks again
